### PR TITLE
disable ltp syslog01 test since rsyslog is disabled by default on F30

### PR DESCRIPTION
**Pull request by rasibley**

### CODE REVIEW

File: distribution/ltp/lite/configs/RHELKT1LITE.20190115 [hunk 1]
```diff
@@ -1233,7 +1233,8 @@ sysfs04 sysfs04
 sysinfo01 sysinfo01
 sysinfo02 sysinfo02
 
-syslog01 syslog01
+# rsyslog is disabled by default on F30
+#syslog01 syslog01
```
Comment (veruu) [inline distribution/ltp/lite/configs/RHELKT1LITE.20190115]: Would it be better to check if we're running on Fedora 30 in the knownissue script and exclude the test that way, so we can still run it on the other releases?

**Comment by rasibley** [comment]
@veruu good point, we do run this version for ark kernels as well using rhel8, I adjusted it, however I probably should test it to make sure I didn't break it :-)

**Comment by veruu** [comment]
This should be a distro thing rather then kernel if I'm not mistaken (and new Fedora should be using systemd's journaling). If the ARK testing will continue to use RHEL8 then we should be ok, I don't remember the test failing there.

**Comment by veruu** [comment]
@rasibley LGTM, let me know if you want to merge it now or you want to do some testing first

**Comment by rasibley** [comment]
@veruu ok it's ready now, just ran a few test jobs and it passed, I had to disable all syslog tests since the others were failing too.

**Pull request merged**